.changeset/quiet-pets-end.md

```diff
@@ -0,0 +1,6 @@
+---
+"@replexica/cli": major
```

Comment:
1. please use either patch or minor
2. please bump both `@replexica/cli` and `replexica` whenever you're planning to bump the CLI (we should make this less confusing)